### ISSUE TITLE
Rename "About" menu nav option to "About us"

### ISSUE THREAD

**Issue by 20BBrown14** [20BBrown14/digital-terrain-mc-js#14]
"About us" I think more exemplifies what will be on this page.

Related: https://github.com/20BBrown14/digital-terrain-mc-js/issues/13

**Comment by juan-colombo95** [comment]
Can I take this issue?

**Comment by 20BBrown14** [comment]
@juan-colombo95 Sure. Can I ask how you found this repo though?

**Comment by juan-colombo95** [comment]
Yes,I search in github for the label: good first issue and  I saw your issue there.

To fix the problem only need change this line :https://github.com/20BBrown14/digital-terrain-mc-js/blob/01b7d27d664ff474026b95e4285b08eb8c4c8028/src/components/header/HeaderView.jsx#L19`
For this : '<Menu.Item key="About">About us</Menu.Item>',right?

**Comment by 20BBrown14** [comment]
Ya. That should be it.

**Comment by juan-colombo95** [comment]
I don't have access for the project.
In the console show this error:
```
$ git push --set-upstream origin Rename_about
remote: Permission to 20BBrown14/digital-terrain-mc-js.git denied to juan-colombo95.
fatal: unable to access 'https://github.com/20BBrown14/digital-terrain-mc-js/': The requested URL returned error: 403
```

**Comment by 20BBrown14** [comment]
Right. You'll need to fork the repo, make your changes on the fork and open a pr from your fork here.

**Comment by juan-colombo95** [comment]
Thank you for your help,i make a pr here: https://github.com/20BBrown14/digital-terrain-mc-js/pull/15
I need to do something else? everything is alright?

**Comment by 20BBrown14** [comment]
Resolved with #15
